src/schema/v2/me/conversation/message.ts

```diff
@@ -86,7 +91,19 @@ export const MessageType = new GraphQLObjectType<any, ResolverContext>({
       description:
         "Unaltered text if possible, otherwise `body`: a parsed/sanitized version from Sendgrid.",
       type: GraphQLString,
-      resolve: ({ body, original_text }) => {
+      resolve: ({
+        body,
+        original_text,
+        conversation_from_name,
+        conversation_initial_message,
+        is_first_message,
+      }) => {
+        if (is_first_message) {
+          const parts = conversation_initial_message.split(
```

Comment:
Is it possible for a conversation's `initial_message` to be `null`?

Just did a quick scan of the impulse schema and it looks like there are no non-null constraints on that column - wondering if that is something we should check here or if it is more of a rare edge case.

Reply:
That's a good question. If it's `null` maybe we should throw an error. In my mind, we _never_ want to show a message if it's got information intended for a partner attached to it.

Reply:
I updated it to throw an exception in that case.

Reply:
cc @starsirius what do you think? Should we do more to solidify that upstream?

Reply:
Looks like we do have some conversations have `null` `initial_message`:
```
Conversation.where(initial_message: nil).count
=> 12610
```
Briefly looked at a few of them, they were created from Eigen. I wonder if Eigen allows submitting inquiries with empty message. 🤔

Reply:
Hmmmm.... I'm not sure what we want to do here then. Unless we try to parse the message for the stuff we send to the gallery partner we're going to end up exposing what we send to them to a collector. Not that that's _terrible_ it's just a bit... awkward?

Reply:
I'm not sure if we want to raise an exception here. Maybe it's ok to just return nothing and let clients handle some kind of fallback?

Reply:
Hmmm... there is no fallback?

Reply:
Yeah when there is no initial message, the email that partners get is just the template with empty message (even the default/auto-populated "Hi, I’m interested in purchasing this work..." message) in it.

Maybe we need some design input about the fallback in this case, but I also think that initial message should be required when collector submits an inquiry.

Reply:
Ahhh, I see. Yes, makes sense.

Reply:
I moved this to be empty as per design's suggestion.

Reply:
Following up w/ a conversation [here](https://artsy.slack.com/archives/C9YNS4X32/p1588085836070400?thread_ts=1588085272.067600&cid=C9YNS4X32)

If the `initial_conversation` is null (or empty), we're going to return null to the client. Eigen will handle this in the same way it'd handle an empty string.